.github/workflows/bump.yml

```
@@ -95,105 +90,25 @@ jobs:
 
       - name: Determine overlays (non-admin)
         id: overlays
+        if: ${{ matrix.doc_id != 'admin' }}
         run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2086:info:2:35: Double quote to prevent globbing and word splitting

(shellcheck)